### ISSUE TITLE
/firefox-ios/Utils/AsyncReducer.swift:6:8: No such module 'Deferred'

### ISSUE THREAD

**Issue by lijieaa** [mozilla-mobile/firefox-ios#2199]
/firefox-ios/Utils/AsyncReducer.swift:6:8: No such module 'Deferred'

**Comment by Hanwp** [comment]
me too

**Comment by JohnJGL** [comment]
me too

**Comment by JohnJGL** [comment]
It's because you haven't install Carthage, try 'brew Install Carthage' in you terminal.

**Comment by JohnJGL** [comment]
That's because you haven't install carthage

**Comment by Hanwp** [comment]
```
bogon:firefox-ios hw$ sh ./bootstrap.sh 
*** Checking out Alamofire at "3.5.0"
*** Checking out Deferred at "7d3d3230133db71544e60dfa7fa3d095361398db"
*** Checking out Fuzi at "0.4.0"
*** Checking out GCDWebServer at "3.3.2"
*** Checking out ios_sdk at "v4.10.2"
*** Checking out KIF at "v3.5.1"
*** Checking out libPhoneNumber-iOS at "0.8.11"
*** Checking out onepassword-extension at "d99f5d7a4c7328f8e3a8d1e7aff455d1072e0c19"
*** Checking out onyx_ios at "0.0.5"
*** Checking out readability at "ac878bf7c15b47d67e7b947584ca108398dd48a4"
*** Checking out SDWebImage at "3.7.4"
A shell task (/usr/bin/env git clone --quiet https://chromium.googlesource.com/webm/libwebpbogon:firefox-ios hw$
```

I executed the `sh ./bootstrap.sh` but the error still exists

**Comment by cuiwe000** [comment]
/Users/XXXX/Documents/Mine_Workspace/firefox-ios/Utils/AsyncReducer.swift:6:8: No such module 'Deferred'

**Comment by welkiner** [comment]
@Hanwp the domain "googlesource.com" was blocked by GFW,you should use proxy.😂

**Comment by Hanwp** [comment]
@welkiner I already used proxy. 😟😟😟And I can visit  "googlesource.com" in Safari Or Chrome.

**Comment by welkiner** [comment]
@Hanwp  proxy for Terminal，not for browser

**Comment by Hanwp** [comment]
@welkiner I used global proxy.

**Comment by Hanwp** [comment]
@welkiner  I was wrong. Just now, I connected VPN, then executed the` sh ./bootstrap.sh`, finally I get it. Thank you. And I want say fuck GFW!

**Comment by 1299681745** [comment]
@Hanwp 你好我想问一下,我在编译项目时,XCGLogger报 Missing required module 'SwiftTryCatch'
